packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItems/FieldGuideItems.js

```diff
@@ -1,19 +1,21 @@
-import { Grid } from 'grommet'
+import { Box, Grid } from 'grommet'
 import React from 'react'
 import PropTypes from 'prop-types'
 import FieldGuideItemAnchor from './FieldGuideItemAnchor'
 
 export default function FieldGuideItems ({ className, items }) {
   return (
-    <Grid
-      className={className}
-      columns={{ count: 'fill', size: '100px' }}
-      gap='medium'
-      rows='150px'
-      width='100%'
-    >
-      {items.map((item, index) => <FieldGuideItemAnchor key={item.title} item={item} itemIndex={index} />)}
-    </Grid>
+    <Box overflow='auto'>
+      <Grid
+        className={className}
```

Comment:
This shouldn't need a `className`, as it's not wrapped in `styled()`

Reply:
This would be broken anyway since I made the outer container the Box.